cinema/views.py

```diff
@@ -167,6 +186,24 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "list", "items": {"type": "number"}},
+                description=("Filter by title id "
+                             "(ex. ?date=2023-09-06,2023-09-07))"),
```

Comment:
invalid description!
what is "Filter by title id"?
should be 'OpenApiTypes.DATE' type